src/content/protocols/sky.md

```diff
@@ -0,0 +1,397 @@
+---
+protocol: "Sky"
+website: "https://sky.money/"
+x: "https://x.com/SkyEcosystem"
+github: ["https://github.com/makerdao"]
+defillama_slug: ["sky-lending"]
+chain: "Ethereum"
+stage: 0
+reasons: []
+risks: ["L", "H", "H", "H", "L"]
+author: ["mmilien_"]
+submission_date: "2025-03-11"
+publish_date: "2025-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Sky is a stablecoin protocol allowing users to mint its USDS stablecoin through _Collateralized Debt Positions_ with a variety of crypto collateral assets. Sky is built on the Maker protocol, by the MakerDAO, to replace the DAI stablecoin and allow for the 1:1 conversion between DAI and USDS. USDS can further be staked for the yield-bearing sUSDS token. Various levels of control over the Sky protocol are exercised through an onchain governance system.
+
+# Ratings
+
+## Chain
+
+This review focuses on the Ethereum deployment of Sky, currently the only chain on which Sky protocol is deployed. Ethereum achieves a Low centralization risk score."
```

Comment:
This review focuses on the Ethereum instance of Sky. Ethereum achieves a Low centralization risk score.